--- git status ---
On branch dev
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   auto_play.sh
	modified:   auto_train.sh

no changes added to commit (use "git add" and/or "git commit -a") 


--- git diff ---
diff --git a/auto_play.sh b/auto_play.sh
index b67e075..442d67c 100755
--- a/auto_play.sh
+++ b/auto_play.sh
@@ -7,7 +7,7 @@ set -euo pipefail
 PROJECT_ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 ISAACLAB_SH="$HOME/IsaacLab/isaaclab.sh"
 source "$HOME/miniconda3/etc/profile.d/conda.sh"
-conda activate sim
+conda activate isaac
 
 # 기본 설정
 DEFAULT_TASK="SpotATS-Velocity-Flat-v0"
@@ -79,4 +79,4 @@ OPTS=(
 [ "$DEFAULT_USE_GUI" = false ] && OPTS+=(--headless)
 
 cd "$PROJECT_ROOT"
-"$ISAACLAB_SH" -p "scripts/rsl_rl/play.py" "${OPTS[@]}"
\ No newline at end of file
+"$ISAACLAB_SH" -p "scripts/rsl_rl/play.py" "${OPTS[@]}"
diff --git a/auto_train.sh b/auto_train.sh
index b0ff411..7442d4f 100755
--- a/auto_train.sh
+++ b/auto_train.sh
@@ -8,14 +8,14 @@ set -euo pipefail
 PROJECT_ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 ISAACLAB_SH="$HOME/IsaacLab/isaaclab.sh"
 source "$HOME/miniconda3/etc/profile.d/conda.sh"
-conda activate sim
+conda activate isaac
 
 # # 실험 식별 (WandB/Folder 명칭)
 EXP_NAME="spot_flat"
 SEED=42
 
 # 실험 타입: baseline / fix
-EXP_VARIANT="fix" 
+EXP_VARIANT="fix"
 
 # #  PARAMETER SETTINGS (여기만 수정하세요)
 GAIT_WEIGHT=14.0
@@ -34,7 +34,7 @@ SAVE_INTERVAL=50
 INITIAL_ITERS=3000
 
 # # 이어하기(Resume) 제어
-RESUME=true              # true: 이어하기 / false: 신규 학습
+RESUME=false              # true: 이어하기 / false: 신규 학습
 RESUME_ITERS=5000       # 추가 학습 횟수 (기존(10000) + 5000 = 15000 종료)
 LOAD_RUN="2026-02-02_10-28-42_gait_w14p0_fix13"     # 불러올 이전 실험 태그
 CHECKPOINT=""            # 특정 pt파일 지정 (비우면 최신 checkpoint 자동 로드)
@@ -56,7 +56,7 @@ OPTS=(
     --seed "$SEED"
     --max_iterations "$CURRENT_MAX_ITERS"
     --num_envs "$NUM_ENVS"
-    --headless
+    # --headless
     # --logger "wandb"            # wandb 바로 연동하고싶을때
     --log_project_name "SpotATS"
 )
@@ -104,4 +104,4 @@ echo ">> Running: $EXP_NAME / $RUN_TAG (Target: $CURRENT_MAX_ITERS iters)"
 # echo ">> [CANCEL] 종료를 취소하려면 1분 이내에 'sudo shutdown -c'를 입력하세요."
 
 # # 1분 후 종료 (실수했을 때 취소할 시간을 벌기 위함)
-# sudo shutdown -h +1
\ No newline at end of file
+# sudo shutdown -h +1